app/security.py

```diff
@@ -78,6 +78,10 @@ def _format_csp(csp_dict):
     'script-src',
```

Comment:
line 68 could be deleted, and also `_format_csp` method I think, or is it for some debug purposes? in which case add a comment to say so

Reply:
Removed code lines - hangover from a long ago change that isn't used any more